src/patternfly/components/Nav/nav-subnav.hbs

```diff
@@ -2,7 +2,6 @@
   {{#if nav-subnav--attribute}}
     {{{nav-subnav--attribute}}}
   {{/if}}
-
   {{#if nav-item--expanded}}
```

Comment:
I think you can use `{{#unless` instead and make this simpler. https://handlebarsjs.com/builtin_helpers.html

Reply:
swapped with unless